### PR TITLE
Resolve Issue 753 Form URLs

### PR DESCRIPTION
**Pull request by Anthony-Calderaro**
In addressing the changes to #753, I performed the following:
- Updated a links to case related documents from the previous endpoint to the one that matches the [forms page](https://www.ustaxcourt.gov/case_related_forms.html) on the public site
- Updated 3 Tax Court links from `http` to `https`

### CODE REVIEW

File: shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+export const StandingPretrialNotice = ({ options, trialInfo }) => {
+  return (
+    <>
+      <PrimaryHeader />
+      <DocketHeader
+        caseCaptionExtension={options.caseCaptionExtension}
+        caseTitle={options.caseTitle}
+        docketNumberWithSuffix={options.docketNumberWithSuffix}
+        h3="Standing Pre Trial Notice"
```
Comment (sutt0n) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx]: I think `Pretrial` is one word here

File: shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
+const React = require('react');
+const { DocketHeader } = require('../components/DocketHeader.jsx');
+const { PrimaryHeader } = require('../components/PrimaryHeader.jsx');
+
+export const StandingPretrialNotice = ({ options, trialInfo }) => {
```
Comment (sutt0n) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx]: This component will have to be added to `documentGenerators.js`, `reactTemplateGenerators.js`, and added to the `getDocumentGenerators()` method in `web-api/applicationContext.js` if you're wanting this to be a generated document. 

CC: @matthopson - I miss anything?

Comment (Anthony-Calderaro) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx]: I messed up my local staging branch so this file was erroneously included. I deleted it to match staging.

File: shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx (hunk 1 above)
Comment (Anthony-Calderaro) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx]: I messed up my local staging branch so this file was erroneously included. I deleted it to match staging.

Although to be fair, I wasn't the one who misspelled this 😁

**Review by mmarcotte — APPROVED** [review]

**Comment by mmarcotte** [comment]
forms

Looks great! Thanks for scanning for the `http://` links, too!

**Pull request merged**